src/amo/actions/landing.js

```diff
@@ -0,0 +1,29 @@
+import {
+  LANDING_GET,
+  LANDING_LOADED,
+  LANDING_FAILED,
+} from 'core/constants';
+
+
+export function getLanding({ addonType }) {
+  return {
+    type: LANDING_GET,
+    payload: { addonType },
+  };
+}
+
+export function loadLanding(
+  { addonType, entities, featured, highlyRated, popular }
```

Comment:
It would be nice to do some payload checking here:
* make sure `addonType` is set
* make sure at least one of the `featured`, `highlyRated`, or `popular` keys are set
* check for `result`, and `entities` within a set key

I don't know if that's overkill but it seems like it might help prevent typo'd calls.

I want to land this soon: https://github.com/mozilla/addons-frontend/issues/1461